src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -32,37 +33,39 @@
 @end
 
 @snippet sampleCode(apiMethod)
-    @if apiMethod.isLongRunningOperation
-        {@longRunningMethodSampleCode(apiMethod)}
-    @else
-      @switch apiMethod.grpcStreamingType
-      @case "NonStreaming"
-          @switch apiMethod.type
-          @case "OptionalArrayMethod"
-              {@methodSampleCode(apiMethod)}
-          @case "PagedOptionalArrayMethod"
-              {@pagedMethodSampleCode(apiMethod)}
-          @default
-              $unhandledCase: {@apiMethod.type}$
-          @end
-      @case "ClientStreaming"
-          {@clientStreamingMethodSampleCode(apiMethod)}
-      @case "ServerStreaming"
-          {@serverStreamingMethodSampleCode(apiMethod)}
-      @case "BidiStreaming"
-          {@bidiStreamingMethodSampleCode(apiMethod)}
-      @default
-          $unhandledCase: {@apiMethod.grpcStreamingType}$
-      @end
+    {@incodeSamples(apiMethod)}
+@end
 
+@snippet incodeSamples(apiMethod)
+    @join sample : apiMethod.samples on BREAK.add(BREAK).add(BREAK).add("// Alternatively:").add(BREAK)
+        {@incodeSampleCall(apiMethod, sample)}
     @end
 @end
 
-@private methodSampleCode(apiMethod)
-    @if apiMethod.hasReturnValue
-        $response = {@methodCallSampleCode(apiMethod)};
-    @else
-        {@methodCallSampleCode(apiMethod)};
+@private incodeSampleCall(apiMethod, sample)
```

Comment:
Add a comment saying the structure of this should be parallel to the method in standalone_sample.snip. And add a comment there to make sure the structure should be parallel to the one here.

(See py for an example)